src/pages/issues/index.tsx

```diff
@@ -25,7 +25,13 @@ const SearchField = ({
     loading,
 }: SearchFieldProps) => {
     return (
-        <div className={classNames.searchFieldContainer}>
+        <form
+            className={classNames.searchFieldContainer}
+            onSubmit={(e) => {
+                e.preventDefault();
+                onSearchTextSubmitted();
+            }}
```

Comment:
pass `onSearchTextSubmitted` as callback and prevent default there.

Reply:
This function is pass from props the changing the function will change the behaviour at multiple location.